### PR TITLE
docs: document what are possible values of format option

### PR DESCRIPTION
**Pull request by big-kahuna-burger**
## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [x] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This action can be used to deploy your authorization model to an OpenFGA store.
 | `store-id` | Required | The store to which the model should be deployed     |
 | `model-file-path` | Optional | The path to your model file relative to the root of your project     |
 | `model` | Optional | The model value if file is not used, Ensure it is character escaped.     |
-| `format` | Optional | The format of the authorization model     |
+| `format` | Optional | Authorization model input format. Can be "fga" or "json"     |
```
Comment (rhamzeh) [inline README.md]:
```suggestion
| `format` | Optional | Authorization model input format. Can be "fga" or "json", defaults to auto-detecting from the file extension |
```

File: action.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ inputs:
     description: The model to be imported into. The characters should be escaped. The value from here will be used if model-file-path is empty
     required: false
   format:
-    description: The format of the Authorization Model
+    description: Authorization model input format. Can be "fga" or "json"
```
Comment (rhamzeh) [inline action.yml]:
```suggestion
    description: Authorization model input format. Can be "fga" or "json", defaults to auto-detecting from the file extension 
```

**Review by rhamzeh — COMMENTED** [review]
Thanks @big-kahuna-burger! Suggested a small change to mention the defaults

**Review by rhamzeh — APPROVED** [review]

**Comment by rhamzeh** [comment]
Approved, but converting to draft so that we do not accidentally merge. We need to enable the Linux Foundation's EasyCLA bot on this repo before we can merge PRs.

Will ping you once we have done so!

**Comment by big-kahuna-burger** [comment]
Ok thanks

**Comment by rhamzeh** [comment]
@big-kahuna-burger mind signing the CLA above? Once it is signed, we should be good to merge!

**Comment by big-kahuna-burger** [comment]
Done 😊

**Comment by rhamzeh** [comment]
Thanks!

**Pull request merged**